.rubocop.yml

```diff
@@ -78,43 +83,25 @@ Style/ClassAndModuleChildren:
 Metrics/ClassLength:
   Enabled: false
 
-Layout/IndentationWidth:
-  Enabled: false
-
 Layout/EndAlignment:
-  Enabled: false
-
-Layout/ElseAlignment:
-  Enabled: false
-
-Performance/ChainArrayAllocation:
-  Enabled: true
+  EnforcedStyleAlignWith: start_of_line
```

Comment:
We didn't know this cop was dis activated! We would like to just activate this cop with the default setting if it is alright because the assignments are more visible (especially if they are hidden inside an algorithmic logic)

If the variable name and the condition is short enough, it is very clear
```ruby
@clusters[index].data_items.each{ |d_i|
  distance_weight = if d_i[4][:moved_up]
                      0.1
                    elsif d_i[4][:moved_down]
                      10
                    else
                      1
                    end
}
```

If the variable name is too long and/or the condition is too long we find the following assignment more visible
```ruby
clusters.each_with_index{ |cluster, index|
  data_sticky = cluster.data_items.collect{ |data| data[4] }.compact
  data_skills = cluster.data_items.collect{ |data| data[5] }.compact
  sub_problem[index][:vrp][:vehicles] =
    if !data_sticky.empty?
      [vehicle_list.find{ |vehicle| vehicle[:id] == data_sticky.flatten.first }].compact
    elsif data_sticky.empty? && !data_skills.empty? && vehicle_list.any?{ |vehicle| vehicle[:skills] && !vehicle[:skills].first.empty? }
      [vehicle_list.find{ |vehicle| vehicle[:skills] && !vehicle[:skills].first.empty? && ([data_skills.flatten.first] & vehicle[:skills].first).size > 0 }]
    else
      [vehicle_list.last] # TODO : function that return the best vehicle for this cluster
    end
  vehicle_list -= sub_problem[index][:vrp][:vehicles]
}
```

instead of the following because the if condition and the assignment is lost in the clutter and the logic is not visible immediately:

```ruby
clusters.each_with_index{ |cluster, index|
  data_sticky = cluster.data_items.collect{ |data| data[4] }.compact
  data_skills = cluster.data_items.collect{ |data| data[5] }.compact
  sub_problem[index][:vrp][:vehicles] = if !data_sticky.empty?
    [vehicle_list.find{ |vehicle| vehicle[:id] == data_sticky.flatten.first }].compact
  elsif data_sticky.empty? && !data_skills.empty? && vehicle_list.any?{ |vehicle| vehicle[:skills] && !vehicle[:skills].first.empty? }
    [vehicle_list.find{ |vehicle| vehicle[:skills] && !vehicle[:skills].first.empty? && ([data_skills.flatten.first] & vehicle[:skills].first).size > 0 }]
  else
    [vehicle_list.last] # TODO : function that return the best vehicle for this cluster
  end
  vehicle_list -= sub_problem[index][:vrp][:vehicles]
}
```

Reply:
The second block is the most elegant, whatever value of `EnforcedStyleAlignWith` it's authorized, this cop is not very relevant